e2e/utils/notification.ts

```diff
@@ -0,0 +1,26 @@
+import { Page } from '@playwright/test';
+
+type NotificationType =
+  | 'reject'
+  | 'approve'
+  | 'revoke'
+  | 'approve-error'
+  | 'create-strategy'
+  | 'renew-strategy'
+  | 'edit-strategy-name'
+  | 'withdraw-strategy'
+  | 'delete-strategy'
+  | 'change-rates-strategy'
+  | 'trade';
+
+export class NotificationDriver {
+  private notif = this.page.getByTestId(`notification-${this.type}`);
```

Comment:
can this be called before the constructor??

Reply:
No this is some typescript sugar, in this code will be compiled into the constructor